custom_components/zwave_mqtt/climate.py

```diff
@@ -460,6 +489,6 @@ def _mode(self) -> None:
 
     def _current_mode_setpoints(self) -> Tuple:
         """Return a tuple of current setpoint Z-Wave value(s)."""
-        current_mode = str(self.values.primary.value).lower()
+        current_mode = self.values.primary.value["Selected_id"]
```

Comment:
``VALUE_SELECTED`` already exists as a constant for ``Selected_id``

Reply:
thanks!
BTW: I assume you own the other type of thermostat yourself ?
This "multi setpoint" type of thermostat was broken...

Reply:
I've never used the original ZW component and didn't know they had merged heating and cooling together. I thought the way it was working for me was the expected behavior with the separate entities

Reply:
No problem, this PR will fix it.